public/src/components/App.jsx

```diff
@@ -85,6 +52,7 @@ function App() {
     }, [discount]);
 
 //HANDLING CHECKIN AND CHECKOUT
+
   function updateCheckInOut(date, checkIn, checkOut) {
     const dateString = date.toString().split(' ').slice(0, 4).join(' ');
```

Comment:
I will look into the necessity of this format. I did it because of the way that I use checkIn and checkOut values in rendering CheckIn, CheckOut and Calendar components, but there might be a more efficient way to do it. If not, I will write the function.